Sources/SaferContinuation/SaferContinuation.swift

```diff
@@ -51,9 +51,22 @@ final public class SaferContinuation<C: Continuation & Sendable>: @unchecked Sen
 	  - context: Allows you to provide any arbitrary data to differentiate between different continuations that you can inspect when errors are thrown or
 	 notifications posted.. Could be a string, a UUID, a UIImage, or your mom's nickname. The last one is probably not useful though. You be the judge.
 	 */
-	public init(_ continuation: C, isFatal: SaferContinuation<UnsafeContinuation<Void, Error>>.FatalityOptions = false, timeout: TimeInterval? = nil, delayCheckInterval: TimeInterval? = 3, file: StaticString = #file, line: Int = #line, function: StaticString = #function, context: Any? = nil) {
+	public init(_ continuation: C,
+                isFatal: SaferContinuation<UnsafeContinuation<Void, Error>>.FatalityOptions? = nil,
+                timeout: TimeInterval? = nil,
+                delayCheckInterval: TimeInterval? = 3,
+                file: StaticString = #file,
+                line: Int = #line, 
+                function: StaticString = #function,
+                context: Any? = nil) {
```

Comment:
(this comment applies to the rest of the PR as well) this is a tabs indentation project. Please fix the indentation to reflect as such.

Also, please place a newline before `_` and after `nil` so the resulting indentation looks like

```swift
	func init(
		_ continuation: C,

		...

		context: Any? = nil
	) {
	// implementation

	}
```

I need to write up a style guide, so I apologize for this style not being apparent earlier.

Reply:
done.